iosHyperskillApp/iosHyperskillApp/Sources/Models/Constants/Images/Images.swift

```diff
@@ -76,6 +76,11 @@ enum Images {
         static let projectGraduate = "track-project-graduate"
         static let hammer = "track-hammer"
 
+        enum TopicsToDiscoverNext {
+            static let completedTopic = "track-topics-to-discover-next-completed-topic"
```

Comment:
I think you can use `Image(systemName: "checkmark")` instead of adding new resource for that